app/services/form_data/data_store.rb

```diff
@@ -32,7 +33,20 @@ def get_date(key)
     end
   end
 
+  def clean
+    @session[@form_key] = {}
+  end
+
   def destroy
     @session.delete(@form_key)
   end
+
+  def to_s
```

Comment:
I can see any references to this method. Is it needed?

Reply:
Not sure where it is used.
Maybe Tony needed it for debugging purposes?

Reply:
Or what is me?
Such a long time that I hardly remember.
Let's remove it. We'll find out if it is needed somewhere.